mixins/labelled-mixin.js

```diff
@@ -93,8 +93,8 @@ export const LabelledMixin = superclass => class extends superclass {
 
 	constructor() {
 		super();
+		this.labelRequired = true;
 		this._labelElem = null;
-		this._labelRequired = true;
```

Comment:
I made this property slightly more public so that components can set it... but it's still not a Lit property so I don't think we need to advertise it.